### DISCUSSION TITLE
开源之道每周评论 2021 08 15 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#63] [Announcements]
# 开源之道每周评论 2021 08 15 - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2021/%E5%BC%80%E6%BA%90%E4%B9%8B%E9%81%93%E6%AF%8F%E5%91%A8%E8%AF%84%E8%AE%BA-2021-08-15/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

![](https://ipic.zoomquiet.top/2022-06-10-WechatIMG1076.jpeg)

有bug...
